ui/scripts/cloudStack.js

```diff
@@ -117,22 +117,15 @@
                         }
                         return cookieValue;
                     };
-                    unBoxCookieValue('sessionkey');
-                    // if sessionkey cookie exists use this to set g_sessionKey
```

Comment:
The original issue was related to security where session key was stored in cookie; the crux of the fix by Jessica was to use the cookie to set the sessionkey on window (g_sessionKey variable) and destroy it so it may not be read later.